.pre-commit-config.yaml

```diff
@@ -6,16 +6,16 @@
 exclude: '^(\.tox|ci/templates|\.bumpversion\.cfg)(/|$)'
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: master
+    rev: v5.0.0
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: debug-statements
   - repo: https://github.com/timothycrosley/isort
-    rev: master
+    rev: 5.13.2
     hooks:
       - id: isort
-  - repo: https://gitlab.com/pycqa/flake8
-    rev: master
+  - repo: https://github.com/pycqa/flake8
+    rev: 7.1.1
     hooks:
-      - id: flake8
\ No newline at end of file
+      - id: flake8
```

Comment:
None of these are actually run, but pointing to a branch is taboo so as least that's updated